subsys/bluetooth/host/cs.c

```diff
@@ -326,4 +334,134 @@ int bt_cs_start_test(const struct bt_cs_test_param *params)
 	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_TEST, buf, NULL);
 }
 
+void bt_hci_le_cs_subevent_result(struct net_buf *buf)
+{
+	struct bt_conn *conn = NULL;
+	struct bt_hci_evt_le_cs_subevent_result *evt;
+	struct bt_conn_le_cs_subevent_result result;
+	struct net_buf_simple step_data_buf;
+
+	if (buf->len < sizeof(*evt)) {
+		LOG_ERR("Unexpected end of buffer");
+		return;
+	}
+
+	evt = net_buf_pull_mem(buf, sizeof(*evt));
+
+	if (evt->subevent_done_status == BT_HCI_LE_CS_SUBEVENT_DONE_STATUS_PARTIAL) {
```

Comment:
Not too important at this stage but I think this should be moved to below line 367